src/Billable.php

```diff
@@ -35,26 +35,26 @@ public function storeCard(PaymentCard $paymentCard)
         $store->fromPaymentCard($paymentCard);
         $result = $store->process();
 
-        $card = new PaymentCard();
-        $card->fromStore($store);
-        $this->cards()->save($card);
+        $paymentCard->fromStore($store);
+        $this->cards()->save($paymentCard);
 
         return $result;
     }
 
     /**
      * @param string $token
+     * @param PaymentCard|null $paymentCard
      * @return ResponseJson|string
      */
-    public function storeCardByToken($token)
+    public function storeCardByToken($token, PaymentCard $paymentCard = null)
```

Comment:
I think maybe instead of having line 55, you can just do `PaymentCard $paymentCard = new PaymentCard()` in the function parameters here?

Reply:
Maybe not, but I don't see a reason why not. Can you test?